src/change_stream/mod.rs

```diff
@@ -95,7 +99,7 @@ pub struct ChangeStream {
 
     /// The read preference for the initial `$changeStream` aggregation, used for server selection
```

Comment:
As I mentioned in my first comment about this, "read preference" does need to be replaced with "SelectionCriteria"; the rest of the comment should remain the same

Reply:
fixed